sources/mysql/mysqldump_test.go

```diff
@@ -728,7 +729,7 @@ CREATE TABLE test (a text PRIMARY KEY, b text);`,
 	INSERT INTO test (id, a, b, c) VALUES (1,'2019-10-29',4.444,5.44444);
 	`,
 			expectedData: []spannerData{
-				spannerData{table: "test", cols: []string{"id", "a", "b", "c"}, vals: []interface{}{int64(1), getDate("2019-10-29"), float64(4.444), "5.444440000"}}},
+				spannerData{table: "test", cols: []string{"id", "a", "b", "c"}, vals: []interface{}{int64(1), getDate("2019-10-29"), float64(4.444), big.NewRat(136111, 25000)}}},
```

Comment:
I have resolved the TODO at https://github.com/cloudspannerecosystem/harbourbridge/blob/c44ff08a43ce2b36c7d6c511b80ca0c8a0ed4697/sources/postgres/data.go#L189 by returning big.Rat instead of String for Decimal/Numeric types to Go client library. Thus, the test case is updated appropriately.